Src/Plugins/Security/SEC101.SecurePlaintextSecrets.json

```diff
@@ -354,6 +354,12 @@
           "ContentsRegex": "$SEC101/042.DynatraceToken",
           "MessageArguments": { "secretKind": "Dynatrace Key" }
         },
+        {
+          "Id": "SEC101/043",
```

Comment:
Should we merge this with rule 31?

Reply:
It's a good question. They are distinct secrets in the NuGet eco-system and the help for each kind would be distinct. This is my usual test for whether a rule stands on its own, so let's keep these distinct for now.

---
In reply to: [610962329](https://github.com/microsoft/sarif-pattern-matcher/pull/372#discussion_r610962329) [](ancestors = 610962329)

Reply:
Rule 31 looked weird, it was one of the validators with a "refine" id in the regex and no dynamic validation, so I thought maybe it was incomplete/a stub.  Sounds like that's not the case, so taking no action!

---
In reply to: [611238190](https://github.com/microsoft/sarif-pattern-matcher/pull/372#discussion_r611238190) [](ancestors = 611238190,610962329)